docs/UserGuide.md

```diff
@@ -552,52 +585,56 @@ Hence, you can type in the command `list` and and press <kbd>Enter</kbd>.
 
 #### Linking a student to a class : `link`
 
-You can link a student to a class using this command.
+You can organise students into their classes using this command.
```

Comment:
Not sure if organise is a good word to use here as technically, all they can do is link...